packages/react-components/react-charts-preview/library/src/components/CommonComponents/CartesianChart.types.ts

```diff
@@ -412,6 +412,19 @@ export interface ICartesianChartProps {
    * @default true
    */
   useUTC?: string | boolean;
+
+  /**
+   * Enables the chart to automatically adjust its size based on the container's dimensions.
+   * @default false
```

Comment:
make default to be true

Reply:
It doesn't work well when enableReflow prop is true. The chart gets truncated below min width but scrollbar doesn't appear because svg width is set to 100%.
Will have to disable reflow in all examples. Should I apply this change?

Reply:
do you think this is better than enable reflow solution.

Reply:
yes, this solution is better if we fix all the overlapping issues.

Reply:
So, here the bars ( in VBC ) or lines (in LineChart) will get overlapped if we decrease chart size beyond a certain limit ?

Reply:
For bars, we have already fixed the overlapping issue and lines dont have this issue. Currently, only the x-axis tick labels overlap in smaller viewports.

Reply:
set responsive prop to true by default and disabled reflow in examples